### ISSUE TITLE
Option to disable 3g seperetely

### ISSUE THREAD

**Issue by gsture** [GrapheneOS/os-issue-tracker#3952]
I would like to suggest an extra option for configuring network types. At the moment there are options for 4g only and 2g can be disabled but it is not possible to disable 3g. I think an option to only use 4g+5g would be useful to reduce attack surface but also leave 5g enabled instead of only using 4g.

**Comment by flawedworld** [comment]
So, a few thoughts from me.

I can see the attraction, but the reality is that the 5G implementation has still got much attack surface within it, in particular network slicing related code in the modem firmware. I think that this may well be desired if you just want to avoid pre-LTE networks. 

Though, I expect LTE only mode to continue to be the least attack surface for many years, and unless Google begin making their own modems and firmware written in a safe language with actually meaningful exploit mitigations, this is not going to change.

**Comment by gsture** [comment]
> So, a few thoughts from me.
> 
> I can see the attraction, but the reality is that the 5G implementation has still got much attack surface within it, in particular network slicing related code in the modem firmware. I think that this may well be desired if you just want to avoid pre-LTE networks.
> 
> Though, I expect LTE only mode to continue to be the least attack surface for many years, and unless Google begin making their own modems and firmware written in a safe language with actually meaningful exploit mitigations, this is not going to change.

I agree but it it does not matter for this request. Having 3g disabled is still better then having it enabled while not using it. For users that want to use 5g there is currently no seperate option to disable 3g. You have to activate both 5g and 3g or stick with 4g, that makes no sense.
